### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
Take a look please @jordimas @darccio @surfdude29 @rortan134

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -8594,15 +8594,15 @@ msgstr "Videojocs"
 
 #: src/view/com/util/post-embeds/VideoEmbed.web.tsx:163
 msgid "Video not found."
-msgstr ""
+msgstr "No s'ha trobat el vídeo"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:8597]:
```suggestion
msgstr "No s'ha trobat el vídeo."
```
The only very tiny thing that I – a non-Catalan speaker, of course 😅 – spotted is a missing `.` at the end here

Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Why I keep missing them haunts me. Can you tell me what program/utility do you use to spot the errors?
Thanks again for your useful contributions

Comment (surfdude29) [reply src/locale/locales/ca/messages.po]: It's only a very tiny thing, I wasn't even sure if it was worth pointing out tbh 😅

I don't use a program, just my eyes 👀🤷🏻‍♂️

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -4974,7 +4974,7 @@ msgstr "Només pot tenir lletres, nombres i guionets"
 
 #: src/view/com/composer/videos/SubtitleFilePicker.tsx:31
 msgid "Only WebVTT (.vtt) files are supported"
-msgstr ""
+msgstr "Només s'admeten fitxers WebVTT (.vtt)"
```
Comment (rortan134) [inline src/locale/locales/ca/messages.po:4977]:
```suggestion
msgstr "Només s'admeten arxius WebVTT (.vtt)"
```

File: src/locale/locales/ca/messages.po [hunk 3]
```diff
@@ -6627,7 +6627,7 @@ msgstr "Selecciona l'opció {i} de {numItems}"
 
 #: src/view/com/composer/videos/SubtitleFilePicker.tsx:57
 msgid "Select subtitle file (.vtt)"
-msgstr ""
+msgstr "Tria el fitxer de subtítols (.vtt)"
```
Comment (rortan134) [inline src/locale/locales/ca/messages.po:6630]:
```suggestion
msgstr "Tria el arxiu de subtítols (.vtt)"
```

File: src/locale/locales/ca/messages.po [hunk 4]
```diff
@@ -5935,7 +5935,7 @@ msgstr "Elimina la republicació"
 
 #: src/view/com/composer/videos/SubtitleDialog.tsx:260
 msgid "Remove subtitle file"
-msgstr ""
+msgstr "Elimina el fitxer de subtítols"
```
Comment (rortan134) [inline src/locale/locales/ca/messages.po:5938]:
```suggestion
msgstr "Eliminar arxiu de subtítols"
```

Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Perfecte, gràcies @rortan134

**Review by haileyok — APPROVED** [review]
Thanks everyone!

**Pull request merged**